lib/ProductOpener/Products.pm

```diff
@@ -1501,12 +1501,25 @@ sub store_product ($user_id, $product_ref, $comment) {
 
 	$log->debug("store_product - done", {code => $code, product_id => $product_id}) if $log->is_debug();
 
-	my $update_type = $product_ref->{deleted} ? "deleted" : "updated";
+	my $action = "updated";
+
+	if (exists $product_ref->{obsolete}
+		and $product_ref->{obsolete} eq "on")
+	{
+		$action = "archived";
+	}
+	if ($rev == 1) {
+		$action = defined $product_ref->{owner} ? "imported" : "created";
```

Comment:
I don't remember what's the use case for "imported"? Why don't we use "created" even for products created through the producers platform?

Reply:
It was a requirement defined during the brainstorm, and I simply guessed it was about the pro platform. But since we have a flavor field already